docker/run.sh

```diff
@@ -8,27 +8,31 @@ cd "$SCRIPTDIR" || exit 1
 
 source ./env.sh
 
-# Create outputs directory if it does not exist
-[[ -d ./outputs ]] || mkdir ./outputs
-
 echo -e "You are using these values:\n"
+echo -e "Container engine:\t${CONTAINER_ENGINE}"
 echo -e "Volumename:\t${VOLUMENAME}"
 echo -e "Invokeai_tag:\t${CONTAINER_IMAGE}"
 echo -e "local Models:\t${MODELSPATH:-unset}\n"
 
-docker run \
+if [[ "${CONTAINER_ENGINE}" == "podman" ]]; then
+   PODMAN_ARGS="--user=appuser:appuser"
+   unset PLATFORM #causes problems
+fi
+
+"${CONTAINER_ENGINE}" run \
   --interactive \
   --tty \
   --rm \
-  --platform="${PLATFORM}" \
+  ${PLATFORM+--platform="${PLATFORM}"} \
   --name="${REPOSITORY_NAME,,}" \
   --hostname="${REPOSITORY_NAME,,}" \
-  --mount=source="${VOLUMENAME}",target=/data \
```

Comment:
"mount=source=" is that valid syndax?  Podman was confused by it.

Reply:
https://github.com/invoke-ai/InvokeAI/pull/2722#discussion_r1111179189

Was working there, never used Podman 😅

Reply:
<shrug> okay I looked in the documentation and didn't see it... maybe type defaults to volume and the second = is the equivalent of a comma..

Reply:
```
docker run -d \
  --name=nginxtest \
  --mount source=nginx-vol,destination=/usr/share/nginx/html,readonly \
  nginx:latest
```

So replace the equal sign in `--mount=` with a space and you get the same as the [docker docs](https://docs.docker.com/storage/volumes/) are refering to. So can be changed if necesarry.